### DISCUSSION TITLE
Can't install packages

### DISCUSSION THREAD

**Discussion by onderakacik** [ci-group/revolve2#390] [Ask for help]
Hi! 

Following the [installation guide](https://ci-group.github.io/revolve2/installation/) on DAS5 cluster. Worked well till "Install packages" part. When I run "sh student_install.sh" or "sh dev_install.sh" from "/home/username/folder-name/revolve2", I get the error:

Directory './ci_group' is not installable. File 'setup.py' not found.

Any idea how to proceed?

**Comment by surgura** [comment]
Update pip and setuptools packages, then it should work. They are outdated.
